pkg/config/config.go

```diff
@@ -48,12 +54,117 @@ func FromData(yamlData string) (*GithubConfig, error) {
 	return &conf, nil
 }
 
-func syncSecrets(ctx context.Context,
+func (goa *GitHubOrgActions) syncSecrets(ctx context.Context,
```

Comment:
`syncSecrets` is now a method attached to the GithubXActions.

Reply:
Note: `syncSecrets` on GithubOrgActions and GithubRepoActions will be practically identical but they call different interface methods (and have different logs).
I am not sure if we want to somewhat unify them.